code/src/App.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react'
```

Comment:
very nice short clean perfect app.js

Reply:
Don't know if the section is necessary since you are only returning one element, and there are no classes or styles that would render the section vital :)